app/models/container_image.rb

```diff
@@ -59,5 +59,9 @@ def perform_metadata_scan(ost)
     scan_via_miq_vm(miq_cnt_group, ost)
   end
 
+  def prevent_from_starting
+    ext_management_system.annotate(self, {"manageiq.org/security" => "fail"}) # TODO: handle non openshift providers
```

Comment:
@smarterclayton @deads2k This is the patch the annotates non secure images.
- Which label should we set here?
- Any other actions needed to prevent containers based on the image from starting on a master openshift deployed using openshift-ansible (I would like to test)

cc @simon3z